pkg/gui/controllers/files_controller.go

```diff
@@ -818,6 +820,33 @@ func (self *FilesController) createStashMenu() error {
 	})
 }
 
+func (self *FilesController) createMergeConflictToolMenu() error {
+	return self.c.Menu(types.CreateMenuOptions{
+		Title: self.c.Tr.ViewMergeConflictOptionsTooltip,
+		Items: []*types.MenuItem{
+			{
+				Label:   self.c.Tr.OpenMergeTool,
+				OnPress: self.c.Helpers().WorkingTree.OpenMergeTool,
+				Key:     'm',
+			},
+			{
+				Label: self.c.Tr.CheckoutOurs,
+				OnPress: func() error {
+					return self.c.Git().WorkingTree.CheckoutOurs(self.context().GetSelected().Name())
```

Comment:
This only works for top-level files. You want `GetPath()` instead of `Name()` so that it also works for files in folders.

Reply:
And you should call `self.c.Helpers().MergeAndRebase.CheckMergeOrRebase(err)` after this; this will cause any files that no longer have conflicts to be staged, and if no conflicting files are left, it will prompt to continue the rebase.